build/pluginctl/main.go

```diff
@@ -0,0 +1,147 @@
+// main handles deployment of the plugin to a development server using the Client4 API.
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+const helpText = `
+Usage:
+    pluginctl deploy <plugin id> <bundle path>
+    pluginctl disable <plugin id>
+    pluginctl enable <plugin id>
+    pluginctl reset <plugin id>
+`
+
+func main() {
+	err := pluginctl()
+	if err != nil {
+		fmt.Printf("Failed: %s\n", err.Error())
+		fmt.Print(helpText)
+		os.Exit(1)
+	}
+}
+
+func pluginctl() error {
+	if len(os.Args) < 3 {
+		return errors.New("invalid number of arguments")
+	}
+
+	client, err := getClient()
+	if err != nil {
+		return err
+	}
+
+	switch os.Args[1] {
+	case "deploy":
+		if len(os.Args) < 4 {
+			return errors.New("invalid number of arguments")
+		}
+		return deploy(client, os.Args[2], os.Args[3])
+	case "disable":
+		return disablePlugin(client, os.Args[2])
+	case "enable":
+		return enablePlugin(client, os.Args[2])
+	case "reset":
+		return resetPlugin(client, os.Args[2])
+	default:
+		return errors.New("invalid second argument")
+	}
+}
+
+func getClient() (*model.Client4, error) {
```

Comment:
https://github.com/mattermost/mattermost-plugin-starter-template/commit/4523a0a25eb0f34604cefac8a5e588d5450470f5 is quite helpful to have. Could you please also include it?